__tests__/app.test.js

```diff
@@ -100,3 +100,39 @@ describe('api/articles/:article_id',()=>{
         })
     })
 })
+describe('/api/article',()=>{
+    test('GET:200 should return an array of article objects with the correct properties',()=>{
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body:{articles}})=>{
+            articles.forEach((article)=>{
+                expect(article).toEqual(
+                    expect.objectContaining({
+                        article_id: expect.any(Number),
+                        title: expect.any(String),
+                        topic:expect.any(String),
+                        author:expect.any(String),
+                        created_at:expect.any(String),
+                        votes:expect.any(Number),
+                        article_img_url:expect.any(String),
+                        comment_count: expect.any(Number)
+                    })
+                )
+            })
+        })
+    })
+    test('GET:404 when comments data has no data should respond with a status of 404 and an object returned with a msg invalid query',()=>{
+        return db.query('DELETE FROM comments;').then(()=>{
+            return request(app)
```

Comment:
Not sure what the intention is of this test here? If we delete all of the comments from every article when try and get all articles it should error?
I think it should be fine that we might want to see the articles even if there were no comments on them. What if an article has only just been posted and doesn't have any comments on it yet? How would I ever be able to see it to be able to leave a comment?